indra/newview/skins/default/xui/en/emoji_groups.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" ?>
```

Comment:
I feel like this is going to cause localization issues. this differes significantly from our normal 'name+label' structure and likely won't be picked up.

Reply:
In case of such issues the xml file and the kiading algorithm could be transformed with no affect to other UI code

Reply:
This is totally not going to work for localization.

BTW: The checkmark for LLMenuItemCheckGL is still rendered as Emoji and looks ugly!

Reply:
> This is totally not going to work for localization.

There was an intermediate solution without localization files
Now it works for all languages except Turkish